app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class LikesController < ApplicationController
+  def create
+    @likeable = params[:likeable_type].constantize.find(params[:likeable_id])
+    @like = @likeable.likes.create!(user_id: current_user.id)
+    respond_to_format
```

Comment:
No need to define this as an instance variable if you won't use it on the response.